x-pack/plugins/spaces/server/saved_objects/saved_objects_service.ts

```diff
@@ -54,6 +54,16 @@ export class SpacesSavedObjectsService {
             }),
           },
         },
+        2: {
```

Comment:
__question:__ is there any reason we cannot or don't want to define `create` schema here like we defined for `solution`? E.g., `disabledFeatures: schema.maybe(schema.arrayOf(schema.string()))`?

Reply:
We don't change schema, so I thought there is no need to provide it in create. `disabledFeatures` are defined already [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/spaces/server/saved_objects/schemas.ts#L15)

Reply:
Aah, didn't realize that, makes sense, thanks!